coops-ui/src/views/AnnualReport.vue

```diff
@@ -130,16 +128,12 @@
                 </h2>
                 <p>Verify or change your Registered Office Addresses.</p>
               </header>
-              <v-card flat id="AR-step-2-container">
-                <RegisteredOfficeAddress
-                  ref="registeredAddress"
-                  :changeButtonDisabled="!agmDateValid"
-                  :legalEntityNumber="corpNum"
-                  :addresses.sync="addresses"
-                  @modified="officeModifiedEventHandler($event)"
-                  @valid="officeValidEventHandler($event)"
-                ></RegisteredOfficeAddress>
-              </v-card>
+              <RegisteredOfficeAddress
+                :changeButtonDisabled="!agmDateValid"
+                :legalEntityNumber="corpNum"
+                :addresses.sync="addresses"
+                @modified="officeModifiedEventHandler($event)"
+                @valid="officeValidEventHandler($event)" />
```

Comment:
I think I wrote this originally, yuck.

Just a comment: Next time we're in here, I think we don't need the `($event)` on the callbacks. I think Vue knows to call the named method with the obvious parameter.

Reply:
Agreed. And yup, I didn't change it ;)